backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -591,19 +625,37 @@ void SdlGraphicsManager::renderImGui() {
 		_imGuiInited = true;
 	}
 
-	ImGui_ImplOpenGL3_NewFrame();
+#ifdef USE_IMGUI_SDLRENDERER2
+	if (_imGuiSDLRenderer) {
+		ImGui_ImplSDLRenderer2_NewFrame();
+	} else
+#endif
+#ifdef USE_OPENGL
+	{
+		ImGui_ImplOpenGL3_NewFrame();
+	}
+#endif
 	ImGui_ImplSDL2_NewFrame();
 
 	ImGui::NewFrame();
 	_imGuiCallbacks.render();
 	ImGui::Render();
-	ImGui_ImplOpenGL3_RenderDrawData(ImGui::GetDrawData());
+#ifdef USE_IMGUI_SDLRENDERER2
+	if (_imGuiSDLRenderer) {
+		ImGui_ImplSDLRenderer2_RenderDrawData(ImGui::GetDrawData(), _imGuiSDLRenderer);
+	} else
```

Comment:
Same thing here. If `USE_IMGUI_SDLRENDERER2` is defined, but `USE_OPENGL` is not, you will have a dangling `else`. Thus particular spot will lead to the compilation error.